### PR TITLE
Atualização de dependencias

### LINKED ISSUE #1088: Atualização de dependencias
### Contexto

Atualizar as principais dependencias do projeto para possibilitar a utilização de novas ferramentas e evitar possíveis vulnerabilidades de segurança.

### Exemplo

A atualização de ferramentas como `babel`, `webpack`, `eslint` e `react` permitirão a utilização de novas funcionalidades das mesmas que irão diminuir o tempo para o projeto rodar e também simplificar e padronizar algumas coisas na parte de código.

### Requisitos e Critérios de Aceite

- [x] Atualizar as principais dependencias
- [x] Ajustar o código que não está adaptado ao novo padrão

### PR DESCRIPTION
**Pull request by Deivis**
Atualização de dependências gerais com base na atualização do `react-scripts-former-kit-dashboard`.

## Checklist
- [x] Atualização das dependências da Pilot.
- [x] Atualização das dependências do Cockpit.
- [x] Atualizar componentes e configurações afetadas pelas atualizações de depêndencias. 

## Issues linkadas
- [x] Resolves #1088
- [x] Depende do PR https://github.com/pagarme/react-scripts-former-kit-dashboard/pull/25

## Como testar?

Os passos para o teste podem ser os mesmo [deste PR](https://github.com/pagarme/react-scripts-former-kit-dashboard/pull/24) do  `react-scripts-former-kit-dashboard` 

## Notas
- A maioria dos _commits_ são referentes a ajustes de lint
- Os testes precisam ser realizados tanto para Pilot quanto Cockpit
- É necessário testar se na build de produção o React está vindo corretamente de uma _cdn_

### CODE REVIEW

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 3d5cd752621e8ea8d6ff5e84277ed265c282bdca
  * Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 3d5cd752621e8ea8d6ff5e84277ed265c282bdca
  * Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: f8f582acff47602623c73b1696082a4be145eb0d
  * Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: f8f582acff47602623c73b1696082a4be145eb0d
  * Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: f8f582acff47602623c73b1696082a4be145eb0d
  * Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: f8f582acff47602623c73b1696082a4be145eb0d
  * Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Settings/Boleto/Form/index.js [hunk 1]
```diff
@@ -41,22 +41,17 @@ class BoletoForm extends Component {
...
-      t('pages.settings.company.boleto.number')
-    )
-    const isRequired = requiredValidation(
-      t('pages.settings.company.boleto.required')
-    )
-    const daysToAddInExpirationDateLabel =
-      t('pages.settings.company.boleto.daysToAddInExpirationDate.label')
+    const isNumber = numberValidation(t('pages.settings.company.boleto.number'))
+    const isRequired = requiredValidation(t('pages.settings.company.boleto.required'))
+    const daysToAddInExpirationDateLabel = t('pages.settings.company.boleto.daysToAddInExpirationDate.label')
```
[the remark this answers is not in the document]
Comment (Deivis) [reply packages/pilot/src/containers/Settings/Boleto/Form/index.js]: Não vai reclamar por conta das strings gigantes que tem no meio, é para passar assim mesmo 😄

**Comment by Deivis** [comment]
> Notei que o botão de visualizar parcelas deixou de abrir e também não exibe erros no console
> 
> image

Solved 😄

File: packages/pilot/src/pages/LoggedArea/routes.js [hunk 2]
```diff
@@ -2,63 +2,66 @@ import Balance32 from 'emblematic-icons/svg/Extract32.svg'
...
+    path: '/account/settings',
+    title: 'pages.settings.user.menu',
+  },
   anticipation: {
     component: Anticipation,
     exact: true,
     hidden: true,
     icon: Balance32,
-    path: '/anticipation/:id?',
+    path: '/anticipation:id?',
```
Comment (miguelssrs) [inline packages/pilot/src/pages/LoggedArea/routes.js]:
```suggestion
    path: '/anticipation/:id?',
```

File: packages/pilot/src/pages/Transactions/Details/Details.js [hunk 3]
```diff
@@ -135,11 +134,11 @@ const getPaymentCardLabels = t => ({
...
-  very_low: t('pages.transaction.risk_level.very_low'),
+  high: t('pages.transaction.risk_level.high'),
   low: t('pages.transaction.risk_level.low'),
   moderated: t('pages.transaction.risk_level.moderated'),
-  high: t('pages.transaction.risk_level.high'),
   very_high: t('pages.transaction.risk_level.very_high'),
+  very_low: t('pages.transaction.risk_level.very_low'),
 })
 
 // TODO: Remove this function and it usage when the this issue is solved
```
Comment (miguelssrs) [inline packages/pilot/src/pages/Transactions/Details/Details.js]: Acho que esse comentario deveria ser removido

Comment (Deivis) [reply packages/pilot/src/pages/Transactions/Details/Details.js]: Dá não, a issue #681 ainda está aberta 😞

Comment (miguelssrs) [reply packages/pilot/src/pages/Transactions/Details/Details.js]: ah taaaa hahahah

**Comment by Deivis** [comment]
> Testei as seguintes telas, e fora os warnings que reportei antes, só encontrei o detalhe em filtragem de transações:
> 
> * [x]  Lista de transações
>   
>   * Ao selecionar as checkboxes no "Mais Filtros", uma warning é exibida no console após clicar em "Filtrar":
>     image
>     image
> * [x]  Detalhes da transação
> * [x]  Configurações
> * [x]  Extrato
> * [x]  Antecipação
> * [x]  Saque
> * [x]  Redefinição de senha
> * [x]  Cadastro
> * [x]  Login

Vai ser resolvido neste PR do `former-kit`https://github.com/pagarme/former-kit/pull/229

File: packages/pilot/src/containers/Anticipation/Confirmation/index.js [hunk 4]
```diff
@@ -299,11 +299,11 @@ AnticipationConfirmation.propTypes = {
   amount: PropTypes.number.isRequired,
   automaticTransfer: PropTypes.bool.isRequired,
   bankAccount: PropTypes.shape({
-    agencia_dv: PropTypes.string,
     agencia: PropTypes.string,
+    agencia_dv: PropTypes.string,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Anticipation/Confirmation/index.js]: Aqui não seria necessário o `eslint-disable-line camelcase` como feito [nesta linha](https://github.com/pagarme/pilot/pull/1097/files#diff-6f471083b818f6b8ec333e6cfacc4bd3R76)?

File: packages/pilot/src/containers/Anticipation/Confirmation/index.js [hunk 5]
```diff
@@ -299,11 +299,11 @@ AnticipationConfirmation.propTypes = {
...
     bank_code: PropTypes.string,
-    conta_dv: PropTypes.string,
     conta: PropTypes.string,
+    conta_dv: PropTypes.string,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Anticipation/Confirmation/index.js]: Aqui não seria necessário o `eslint-disable-line camelcase`?

File: packages/pilot/src/containers/Refund/BoletoConfirmation/index.js [hunk 6]
```diff
@@ -95,11 +94,11 @@ class BoletoConfirmation extends Component {
 BoletoConfirmation.propTypes = {
   amount: PropTypes.number.isRequired,
   bankAccount: PropTypes.shape({
-    agencia_dv: PropTypes.string,
     agencia: PropTypes.string.isRequired,
+    agencia_dv: PropTypes.string,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Refund/BoletoConfirmation/index.js]: Mesmo ponto do `eslint-disable-line camelcase`

File: packages/pilot/src/containers/Refund/BoletoConfirmation/index.js [hunk 7]
```diff
@@ -95,11 +94,11 @@ class BoletoConfirmation extends Component {
...
     bank_code: PropTypes.string.isRequired,
-    conta_dv: PropTypes.string.isRequired,
     conta: PropTypes.string.isRequired,
+    conta_dv: PropTypes.string.isRequired,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Refund/BoletoConfirmation/index.js]: Mesmo ponto do `eslint-disable-line camelcase`

File: packages/pilot/src/containers/Refund/BoletoForm/index.js [hunk 8]
```diff
@@ -244,11 +244,11 @@ class BoletoForm extends Component {
 BoletoForm.propTypes = {
   amount: PropTypes.number.isRequired,
   bankAccount: PropTypes.shape({
-    agencia_dv: PropTypes.string,
     agencia: PropTypes.string,
+    agencia_dv: PropTypes.string,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Refund/BoletoForm/index.js]: Mesmo ponto do `eslint-disable-line camelcase`

File: packages/pilot/src/containers/Refund/BoletoForm/index.js [hunk 9]
```diff
@@ -244,11 +244,11 @@ class BoletoForm extends Component {
...
     bank_code: PropTypes.string,
-    conta_dv: PropTypes.string,
     conta: PropTypes.string,
+    conta_dv: PropTypes.string,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Refund/BoletoForm/index.js]: Mesmo ponto do `eslint-disable-line camelcase`

**Pull request merged**